### PR TITLE
tf/k8s based deployment

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

**Comment by jhoblitt** [comment]
This deployment is largely based on `lsstsqre/sandbox-eups`.  Per a long ago co-work discussion about people not being happy with `sandbox-` as a prefix, I've used `deploy-`.  Hopefully, we can try to be consistent about repo naming so "deployments" are easy to identify.

I had difficulty choosing a namespace/prefix for k8s.  I wanted to use `sw.lsstcorp.org` but `.` is not a valid char anywhere in a k8s ns.  I went with on `eups-redirect` but don't feel its very intuitive....  I think `sw_lsstcorp_org` would be valid.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+terraform / kubernetes deployment for sw.lsstcorp.org
+===
+
+Redirects from `sw.lsstcorp.org` -> `eups.lsst.codes`
```
Comment (jonathansick) [inline README.md]: I'd suggest a fuller sentence/paragraph. Like:

> Kubernetes deployment of an nginx server that redirects `sw.lsstcorp.org` to `eups.lsst.codes`.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+tl;dr
+---
+
+    bundle install
+    bundle exec rake creds
```
Comment (jonathansick) [inline README.md]: In these docs you'll need to mention the dependency on `~/Dropbox/lsst-sqre/git/lsst-certs.git`.

File: kubernetes/eups-redirect-service.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Load balancer service for Keeper pods
```
Comment (jonathansick) [inline kubernetes/eups-redirect-service.yaml]: Change the names in this comment ;) `Keeper` is in a few other YAML files too.

Comment (jhoblitt) [inline kubernetes/eups-redirect-service.yaml]: Ya, they are in `sandbox-eups` too...

**Comment by jhoblitt** [comment]
@jonathansick Nice catches -- fixed + repushed.

**Pull request merged**